packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -434,11 +437,19 @@ function createApp(tmpDir: string, name: string, parsedArgs: WorkspaceArgs) {
   const command = `new ${name} ${args} --collection=@nrwl/workspace`;
   console.log(command);
 
-  let nxWorkspaceRoot = process.cwd().replace(/\\/g, '/');
-  if (process.platform === 'win32') {
-    nxWorkspaceRoot = `\\"${nxWorkspaceRoot}\\"`;
-  } else {
-    nxWorkspaceRoot = `"${nxWorkspaceRoot}"`;
+  let nxWorkspaceRoot = `"${process.cwd().replace(/\\/g, '/')}"`;
+
+  // If path contains spaces there is a problem in Windows for npm@6.
+  // In this case we have to escape the wrapping quotes.
+  if (
+    process.platform === 'win32' &&
+    /\s/.test(nxWorkspaceRoot) &&
```

Comment:
If It's npm 7 and there are no spaces...

Is there still an issue with the non-escaped quotes?

Reply:
No, NPM 7 works even with spaces.